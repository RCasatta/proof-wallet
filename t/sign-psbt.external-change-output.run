#!/bin/bash
set -e

$GLACIERSCRIPT sign-psbt -m 2 -n 2 --regtest << INPUT 2>&1
y
y
y
y
y
y
mouse scout chest give trim veteran keen arrive test advice warrior reduce silent useful oak embrace drift curve large poet embody theme focus wolf
tpubD6NzVbkrYhZ4YTjZqdGdMpqtd7BFKkERJcK3hjQx4gtHC3xowC5EKAsHY5tgx1coSC6sbw6X7Ngc1AFrVEzgwFFAF8pbsU5n28yYiZrxTAg
tpubD6NzVbkrYhZ4XcWFWZiEr9DBG7eFbG5H6Lz4TwmMw9FwhaRT7Yj5LyPFFBp7jok8uaPKWYsQtHEK2e1N231CmVt2QfMsjCn46by1NbrFEq9
cHNidP8BAKkCAAAAAURrW7a/0IjQJW4qN6xld6NLFTSoGe+WA2VzxApER5svAAAAAAD9////A6CGAQAAAAAAIgAgEe1vpcXX8r5230FYvh3Zs6OS4C5qoXNpbPuSPyuQ1l+eSAIAAAAAACIAINKAw5bQ4w3o4xrs/9qBoYNBK3xTrs1Ce3540CjYC1RqsHELAAAAAAAXqRT/0Nu0RALV+PEtm6W0hKLBu0faQoelfxkAAAEBK0BCDwAAAAAAIgAg5uwMvcznxtBTeTlP1pZI9AY+f6lnhebjWpq2CvesvUgBBUdSIQICL2DXAmD0XKGVlMB3Eg09gDXTogG+FJ2aa+rd3gP3niED9FUZFcYYgC+97g0KJPZH+9zd+5di4ZMKbICeyY3KNIBSriIGAgIvYNcCYPRcoZWUwHcSDT2ANdOiAb4UnZpr6t3eA/eeDDTpPNoAAAAAAAAAACIGA/RVGRXGGIAvve4NCiT2R/vc3fuXYuGTCmyAnsmNyjSADAuIqTcAAAAAAAAAAAABAUdSIQMcqLMnP5sC/KpfXo+CLafbogsbWcHQKxlD9aJRjQZ24CEDZRAYdSaAoQ0vmEDIJHYxEZ+jWLASY3Q+0Jhw9DdgHRtSriICAxyosyc/mwL8ql9ej4Itp9uiCxtZwdArGUP1olGNBnbgDAuIqTcAAAAAAQAAACICA2UQGHUmgKENL5hAyCR2MRGfo1iwEmN0PtCYcPQ3YB0bDDTpPNoAAAAAAQAAAAABAUdSIQJIpi3B16WvnfaxAok7Bj3y3FHgNkdyYmD8fZW9y/evGSECpjgib8VI58A25EhnGrmHPFDlZQcTMaYB/siGkCnQmtRSriICAkimLcHXpa+d9rECiTsGPfLcUeA2R3JiYPx9lb3L968ZDAuIqTcBAAAAAAAAACICAqY4Im/FSOfANuRIZxq5hzxQ5WUHEzGmAf7IhpAp0JrUDDTpPNoBAAAAAAAAAAAA
EXIT
INPUT
