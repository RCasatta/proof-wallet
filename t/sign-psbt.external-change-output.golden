Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
Enter the 24 word mnemonic phrase (separate the words with whitespace): 
Input 2 valid xpubs

Enter xpub #1: 
Enter xpub #2: 
Enter the psbt for the transaction you wish to sign: 
Validating the PSBT...
================================================================================
PSBT validation SUCCESSFUL:
* The provided base64 encoded input is a valid PSBT.
* All input validations succeeded.
* All output validations succeeded.

PSBT validation WARNINGS:
* Tx output 0 spends change to an external receive address. If this is the intended behavior, you can safely ignore this warning.

+-----------------------+
|                       |
|  Transaction Summary  |
|                       |
+-----------------------+
Transaction ID: 98c44e81c09c8bae33e52f017b005e76f1f48f18dc2fbed60c5b0f3278524e8e
Virtual size: 224 vbyte
Fee (total): 0.00000338
Fee (rate): 1.5 sat/byte

Inputs (1)
2f9b47440a...bfb65b6b44:0	bcrt1qumkqe0wvulrdq5me898ad9jg7srrulafv7z7dc66n2mq4aavh4yqk2559z	0.01000000

Outputs (3)
[CHANGE] bcrt1qz8kklfw96letuaklg9vtu8wekw3e9cpwd2shx6tvlwfr72us6e0ssyl0lj	0.00100000
[CHANGE] bcrt1q62qv89ksuvx73cc6anla4qdpsdqjklzn4mx5y7m70rgz3kqt234ql0nnac	0.00149662
[NOT CHANGE] 2NGZrVvZG92qGYqzTLjCAewvPZ7JE8S8VxE	0.00750000

Controls:
    'SIGN' -- sign the psbt
    'EXIT' -- exit

Enter your desired command: Exiting...
